Cargo.lock

```diff
@@ -2081,12 +2047,6 @@ dependencies = [
  "winapi 0.3.8",
 ]
 
-[[package]]
-name = "memoffset"
-version = "0.2.1"
```

Comment:
@RalfJung I'm sure you'll be happy :wink:

Reply:
Yes I am. :-)